ckan/migration/alembic.ini

```diff
@@ -0,0 +1,74 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = .
+
+# template used to generate migration files
+# file_template = %%(rev)s_%%(slug)s
+
+# timezone to use when rendering the date
+# within the migration file as well as the filename.
+# string value is passed to dateutil.tz.gettz()
+# leave blank for localtime
+# timezone =
+
+# max length of characters to apply to the
+# "slug" field
+#truncate_slug_length = 40
+
+# set to 'true' to run the environment during
+# the 'revision' command, regardless of autogenerate
+# revision_environment = false
+
+# set to 'true' to allow .pyc and .pyo files without
+# a source .py file to be detected as revisions in the
+# versions/ directory
+# sourceless = false
+
+# version location specification; this defaults
+# to ckan/migration/alembic/versions.  When using multiple version
+# directories, initial revisions must be specified with --version-path
+# version_locations = %(here)s/bar %(here)s/bat ckan/migration/alembic/versions
+
+# the output encoding used when revision files
+# are written from script.py.mako
+# output_encoding = utf-8
+
+sqlalchemy.url = postgresql://ckan_default:root@localhost/ckan_alembic
```

Comment:
Is this DB actually used? is it a placeholder?

Reply:
Nope, this is default configuration file benerated durinig alembic initialization, that can be used for development purposes and debuging(for example, you can run all alembic commands directly from folder, containing alembic.ini), but CKAN commands(paster, etc.) are initialized with database url from main configuration file. Probably, i should set `user:password` and `dbname` to some base values, like ones from initial configuration file. Or this line may be removed at all...